README.rst

```diff
@@ -162,32 +162,28 @@ Transformer Engine comes preinstalled in the pyTorch container on
 From source
 ^^^^^^^^^^^
 
-First, install the prequisites.
+For JAX and Tensorflow, you need to install that dependency first:
 
 .. code-block:: bash
 
-  apt-get install ninja-build pybind11-dev
```

Comment:
Why is pybind11-dev package not working?

Reply:
It is working. It is just that pip packges works for more OS.
So this is a more general solution.
rpm based OS will work with the pip package.